rpcs3/Emu/RSX/Overlays/HomeMenu/overlay_home_menu.h

```diff
@@ -1,42 +1,32 @@
 #pragma once
 
-#include "overlays.h"
-#include "overlay_list_view.hpp"
+#include "Emu/RSX/Overlays/overlays.h"
 #include "Emu/Cell/ErrorCodes.h"
+#include "overlay_home_menu_main_menu.h"
 
 namespace rsx
 {
 	namespace overlays
 	{
 		struct home_menu_dialog : public user_interface
 		{
-		private:
-			struct home_menu_entry : horizontal_layout
-			{
-			public:
-				home_menu_entry(const std::string& text);
-
-				static constexpr u16 height = 40;
-			};
-
-			std::vector<std::unique_ptr<overlay_element>> m_entries;
-			std::vector<std::function<bool()>> m_callbacks;
-			std::unique_ptr<overlay_element> m_dim_background;
-			std::unique_ptr<list_view> m_list;
-			std::unique_ptr<label> m_description;
-
-			animation_color_interpolate fade_animation;
-
 		public:
-			home_menu_dialog();
+			explicit home_menu_dialog();
```

Comment:
If we actually need this, we have code smell somewhere. Where do we need it?

Reply:
removed